### ISSUE TITLE
Error migrating from 2.23.2 to 2.24.1

### ISSUE THREAD

**Issue by SirMartin** [IgnisDa/ryot#414]
Hi, 

I am just started with the migration to version 3, so following the guide, I update my docker to 2.24.1, but when it tries to make the migrations in the database, it fails, because it cannot find a table.

```2023-10-20T12:31:12.745052Z  INFO ryot: Running version: 2.24.1
2023-10-20T12:31:12.801077Z  INFO ryot: Using database backend: PostgreSQL
2023-10-20T12:31:12.802696Z  INFO sea_orm_migration::migrator: Applying all pending migrations
2023-10-20T12:31:12.806627Z  INFO sea_orm_migration::migrator: Applying migration 'm20231016_create_collection_to_entity'
2023-10-20T12:31:12.848011Z  INFO sea_orm_migration::migrator: Migration 'm20231016_create_collection_to_entity' has been applied
2023-10-20T12:31:12.848684Z  INFO sea_orm_migration::migrator: Applying migration 'm20231016_drop_useless_tables'
Error: Query Error: error returned from database: column collection.last_updated_on does not exist
Caused by:
    error returned from database: column collection.last_updated_on does not exist```

**Comment by IgnisDa** [comment]
Duplicate of #413. Please follow the conversation there.

**Comment by IgnisDa** [comment]
Ah yes this seems to be a bug i introduced. Sorry for that. Can you send me the database backup you made on my email (in my GH profile) so that I can push a fixed docker image?

**Comment by SirMartin** [comment]
Done, thanks!

**Comment by IgnisDa** [comment]
Hey I published `v2.24.2` with a fix for this. Follow the steps in the migration but replace `v2.24.1` with `v2.24.2`. LMK here if anything else comes up.
